src/app/loadout-builder/filter/LockArmorAndPerks.tsx

```diff
@@ -189,14 +176,14 @@ function LockArmorAndPerks({
   return (
     <div>
       <div className={styles.area}>
-        {Boolean(flatLockedMods.length) && (
+        {Boolean(lockedMods.length) && (
           <div className={styles.itemGrid}>
-            {flatLockedMods.map((item) => (
+            {lockedMods.map((mod, index) => (
               <LockedModIcon
-                key={item.key}
-                mod={item.modDef}
+                key={index}
```

Comment:
should never use array index for key, you will be *shocked* at the wild cases where it wrongly decides elements can be re-used

Reply:
forgot to go back and fix this, done